docs/UserGuide.md

```diff
@@ -555,7 +564,7 @@ Just got invited to an interview? Keep your internship role status and other det
 
 **Function:** Edits the specified role in the specified company within the displayed company list.
 
-**Format:** `editRole COMPANY_INDEX ROLE_INDEX [n/ROLE_NAME [(TYPE)]] [s/STATUS] [r/REMINDER_DATE] [d/DESCRIPTION] [$/STIPEND]`
+**Format:** `editRole COMPANY_INDEX ROLE_INDEX n/ROLE_NAME [(TYPE)] s/STATUS [r/REMINDER_DATE] [d/DESCRIPTION] [$/STIPEND]`
```

Comment:
If we are changing the meaning of the brackets here, it must be updated above in the command format to specify what exactly the brackets mean.

Reply:
you are right i added them back.